cypress/e2e/shifting_spec/ShiftingRequest.cy.ts

```diff
@@ -0,0 +1,94 @@
+import LoginPage from "pageobject/Login/LoginPage";
+import { PatientConsultationPage } from "pageobject/Patient/PatientConsultation";
+import { PatientPage } from "pageobject/Patient/PatientCreation";
+import ShiftCreation from "pageobject/Shift/ShiftCreation";
+import { ShiftDetails } from "pageobject/Shift/ShiftDetails";
+import { ShiftingBoard } from "pageobject/Shift/ShiftingBoard";
+import ShiftUpdate from "pageobject/Shift/ShiftUpdate";
+
+describe("New shifting request", () => {
+  const loginPage = new LoginPage();
+  const shiftCreation = new ShiftCreation();
+  const shiftingBoard = new ShiftingBoard();
+  const shiftDetails = new ShiftDetails();
+  const shiftUpdate = new ShiftUpdate();
+  const patientPage = new PatientPage();
+  const patientConsultationPage = new PatientConsultationPage();
+
+  const patientName = "Dummy Patient 16";
+  const currentFacilityPerson = "new";
+  const currentFacilityPhone = "9465666768";
+  const shiftReasonInitial = "emmergency";
+  const ambulanceDriverNameInitial = "Rahul";
+  const ambulancePhoneInitial = "9865666768";
```

Comment:
```suggestion
  const ambulancePhoneInitial = "9865666768";
```

use the generatePhoneNumber() utils function , don't use static phone number

Reply:
change it for all phone numbers